components/security-mgt/org.wso2.carbon.security.mgt/src/main/java/org/wso2/carbon/security/keystore/KeyStoreAdmin.java

```diff
@@ -411,16 +436,20 @@ public void removeCertFromStore(String alias, String keyStoreName)
             if (keyStoreName == null) {
                 throw new SecurityConfigException("Key Store name can't be null");
             }
-
-            KeyStoreManager keyMan = KeyStoreManager.getInstance(tenantId);
-            KeyStore ks = keyMan.getKeyStore(keyStoreName);
+            //TODO:re use keyman when truststore support is added to org.wso2.carbon.core.util.KeyStoreManager
+            //KeyStoreManager keyMan = KeyStoreManager.getInstance(tenantId);
```

Comment:
Remove commented code, if they are required to keep add a comment explaining why.